### ISSUE TITLE
Letter spacing doesn't work with Text

### ISSUE THREAD

**Issue by ashar-7** [JetBrains/compose-jb#82]
```
Text("MARCH", style = MaterialTheme.typography.subtitle2)

// MaterialTheme.typography.subtitle2:
subtitle2 = TextStyle(
    letterSpacing = 2.sp,
    fontSize = 14.sp,
    fontFamily = JosefinSans
)
```
This looks like:
![march](https://user-images.githubusercontent.com/26629623/98689084-927fa100-2391-11eb-828c-a27c5f3f3d49.png)

I'm on windows, not sure if this is the case for other platforms too.

**Comment by olonho** [comment]
Fixed as of 0.2.0-build129.

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6517">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
